Users/Tutorial.rst

```diff
@@ -376,19 +376,20 @@ Show bears' Information
 -----------------------
 
 To get help on using a bear or to get a description of the bear, use the
-``--show-description`` argument:
+``--show-details`` argument:
 
 ::
 
-    coala --bears=SpaceConsistencyBear --show-bears --show-description
+    coala --show-bears --show-details
 
-This will display a large amount of information regarding the bears that
-have been specified (in the ``.coafile`` or in the CLI arguments). It
```

Comment:
`--show-details` and `-show-description` show bear details/descriptions for `--show-bears`